Core/GDCore/Events/CodeGeneration/EffectsCodeGenerator.cpp

```diff
@@ -34,6 +35,15 @@ void ExposeProjectEffects(
       }
     }
   }
+
+  // Add object effects
+  for (std::size_t s = 0; s < project.GetObjectsCount(); s++) {
+    auto& effects = project.GetObject(s).GetEffects();
+    for (std::size_t e = 0; e < effects.GetEffectsCount(); e++) {
+      auto& effect = effects.GetEffect(e);
+      worker(effect);
+    }
+  }

```

Comment:
@4ian this doesn't seem to include (and register) effects. Is something missing here? I'm not sure about what has to be done here.

Reply:
This is only for global objects (i.e: project objects). You also must do this for every layout objects :)